WooCommerce/Classes/ViewRelated/Orders/Order Creation/OrderForm.swift

```diff
@@ -441,78 +466,27 @@ private struct ProductsSection: View {
-                            logPermissionStatus(status: .notPermitted)
-                            self.showPermissionsSheet = true
-                        case .notDetermined:
-                            logPermissionStatus(status: .notDetermined)
-                            viewModel.requestCameraAccess(onCompletion: { isPermissionGranted in
-                                if isPermissionGranted {
-                                    showAddProductViaSKUScanner = true
-                                    logPermissionStatus(status: .permitted)
-                                }
-                            })
-                        case .permitted:
-                            showAddProductViaSKUScanner = true
-                            logPermissionStatus(status: .permitted)
-                        }
-                    }, label: {
-                        if showAddProductViaSKUScannerLoading {
-                            ProgressView()
-                        } else {
-                            Image(uiImage: .scanImage.withRenderingMode(.alwaysTemplate))
-                            .foregroundColor(Color(.brand))
-                        }
-                    })
-                    .sheet(isPresented: $showAddProductViaSKUScanner, onDismiss: {
-                        scroll.scrollTo(addProductViaSKUScannerButton)
-                    }, content: {
-                        ProductSKUInputScannerView(onBarcodeScanned: { detectedBarcode in
-                            showAddProductViaSKUScanner = false
-                            showAddProductViaSKUScannerLoading = true
-                            viewModel.addScannedProductToOrder(barcode: detectedBarcode, onCompletion: { _ in
-                                showAddProductViaSKUScannerLoading = false
-                            }, onRetryRequested: {
-                                showAddProductViaSKUScanner = true
-                            })
-                        })
-                    })
+                    scanProductButton
                     .renderedIf(viewModel.isAddProductToOrderViaSKUScannerEnabled)
                 }
-
-                HStack {
-                    Button(OrderForm.Localization.addCustomAmount) {
-                        showAddCustomAmount.toggle()
-                    }
-                    .accessibilityIdentifier(OrderForm.Accessibility.addCustomAmountIdentifier)
-                    .buttonStyle(PlusButtonStyle())
-                    .sheet(isPresented: $showAddCustomAmount, onDismiss: viewModel.onDismissAddCustomAmountView, content: {
-                        AddCustomAmountView(viewModel: viewModel.addCustomAmountViewModel)
-                    })
-                }
-                .renderedIf(viewModel.shouldShowCustomAmountsWithProducts)
+                .renderedIf(!viewModel.shouldShowCustomAmountsWithProducts || viewModel.productRows.isEmpty)
```

Comment:
Same as above regarding `shoulRenderXYZ` to improve readability.

Reply:
Done in 8828e127417ac49e022c5d321bdf76882367e1c3